docs/deploy/index.md

```diff
@@ -182,31 +186,27 @@ kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/cont
 
 ##### TLS termination in AWS Load Balancer (NLB)
 
-In some scenarios is required to terminate TLS in the Load Balancer and not in the ingress controller.
-
-For this purpose we provide a template:
+By default, TLS is terminated in the ingress controller. But it is also possible to terminate TLS in the Load Balancer. This section explains how to do that on AWS with using an NLB.
```

Comment:
Wondering if LoadBalancer should be one word all over the doc.

Reply:
Good question; I don't know if there is a style guide for the Kubernetes documentation. Personally in my materials I write "load balancer" to indicate a regular load balancer (it could be a cloud LB, HAProxy, whatever), and LoadBalancer when I refer specifically to a Kubernetes Service of type LoadBalancer. (Oh, and I also capitalize ressource names, e.g. Deployment, Pod, Service, PersistentVolumeClaim, so that it matches the kind: field; but it's just my personal convention, I don't know if it's official!)